src/app/base/components/comment/comment.component.html

```diff
@@ -167,14 +170,19 @@
                 style="line-height:1rem;border-radius: 0px" (change)="preventEventPropagation($event)"
                 (keyup)="preventEventPropagation($event)" (keydown)="preventEventPropagation($event)"></textarea>
             <div class="flex flex-row mt-2 justify-between">
-                <button (click)="saveCommentToDb(commentBox.value,privateComment.checked);commentBox.value=''"
-                    class="px-2.5 py-1.5 border border-gray-300 shadow-sm text-xs font-semibold rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none"
-                    [ngClass]="commentBox.value =='' || newComment.commentId == '' ? 'opacity-50 cursor-not-allowed pointer-events-none' : 'opacity-100 cursor-pointer'">
-
+                <button *ngIf="commentBox.value =='' || newComment.commentId == '' else enabledSave"
+                    class="px-2.5 py-1.5 border border-gray-300 opacity-50 cursor-not-allowed pointer-events-none shadow-sm text-xs font-semibold rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none">
                     Save to database
                 </button>
+                <ng-template #enabledSave>
+                    <button (click)="saveCommentToDb(commentBox.value,privateComment.checked);commentBox.value=''"
+                        class="bg-green-100 text-green-700 border border-green-400 text-white text-xs font-semibold px-2.5 py-1.5 rounded-md hover:bg-green-200 focus:outline-none inline-block opacity-100 cursor-pointer">
+                        Save to database
+                    </button>
+                </ng-template>
+
                 <button (click)="newComment.areaOpen=false;"
-                    class="bg-red-100 border border-red-400 text-red-700 text-xs font-semibold px-4 py-2 rounded-md cursor-pointer ml-3 hover:bg-red-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500">
+                    class="bg-red-100 border border-red-400 text-red-700 text-xs font-semibold px-2.5 py-1.5 rounded-md cursor-pointer ml-3 hover:bg-red-200 focus:outline-none">
```

Comment:
I think this buttons have less height than the others, is there any specific reason or should we change them so they will be consistent with the others?

Reply:
to have it a bit more compact but we can change it to the common one :)

Reply:
Changed